test/types/Model.ts

```diff
@@ -9,3 +10,30 @@ const shouldFailWithNumberAsName = dynamoose.model(1);
 const shouldFailWithBooleanAsName = dynamoose.model(true);
 
 const shouldSucceedWithOnlyPassingInName = dynamoose.model("User");
+
+interface User extends Document {
```

Comment:
Trying to figure out how to best format this. Maybe something like the following? To make it clear it's a new section.

```suggestion
// Typed Models

interface User extends Document {
```

Reply:
yes ok, since this is not the finally, we can always go back and review them again.